dashboard/pages/api/api.ts

```diff
@@ -15,8 +15,34 @@
  *
  */
 
+const PROD_API_ENDPOINT = "/api"
+const MOCK_API_ENDPOINT = "http://localhost:32333"
+const EXTERNAL_META_NODE_API_ENDPOINT = "http://localhost:5691/api"
+
+export const PREDEFINED_API_ENDPOINTS = [
+  PROD_API_ENDPOINT,
+  MOCK_API_ENDPOINT,
+  EXTERNAL_META_NODE_API_ENDPOINT,
+]
+
+export const DEFAULT_API_ENDPOINT: string =
+  process.env.NODE_ENV === "production" ? PROD_API_ENDPOINT : MOCK_API_ENDPOINT
```

Comment:
By default we still use the `mock_server` for development and `/api` for production. Developers may set it to `http://localhost:5691/api` so that they can develop dashboard with a real local cluster.